ssp

```diff
@@ -6498,50 +6505,37 @@ sub check_for_suphp_but_no_fileprotect {
 }
 
 sub check_if_hostname_missing_from_localdomains {
-    my $hostname_check = 0;
-
     if ( open my $localdomains_fh, '<', '/etc/localdomains' ) {
         while (<$localdomains_fh>) {
             if (/^${hostname}$/) {
-                $hostname_check = 1;
+                print_warn('Hostname: ');
```

Comment:
This is not equivalent to the original logic.  As it stands if it finds the hostname it will print the warning which is the opposite.  It needs to search the entire file for the hostname and only print the warning if it is not found.

Reply:
good catch, will fix

Reply:
done, pushed